drivers/dma/bcm2835-dma.c

```diff
@@ -818,20 +818,25 @@ static enum dma_status bcm2835_dma_tx_status(struct dma_chan *chan,
 		struct bcm2835_desc *d = c->desc;
 		dma_addr_t pos;
 
-		if (d->dir == DMA_MEM_TO_DEV && c->is_40bit_channel)
```

Comment:
Commit text could be expanded, and would normally reference bcm2835-dma rather than just "dma".